### PR TITLE
Bugfix/Fixing text overflow issues in table for long text

### LINKED ISSUE #1945: Long text overflow from cell in table widget
### What is the expected behaviour?
text to be wrapped or be scrollable in cell
new option to be added to include scroll, text wrapping so that text does not overflow.

### What is the current behaviour?
long text / sentences overflows in table.

### How to reproduce the issue?

1.drag and drop table component
2.write long  text in any of the fields 
3.text can be seen overflowing from the table cell

### Screenshots or Screencast
Screenshot 2022-01-26 at 1 51 54 AM

### Any possible solutions?

option/property can be included as a dropdown to set behaviour of text with options like wrap , scroll etc.

### PR DESCRIPTION
**Pull request by stepinfwd**
closes #1945

### CODE REVIEW

File: frontend/src/Editor/Inspector/Components/Table.jsx [hunk 1]
```diff
@@ -544,6 +561,8 @@ class Table extends React.Component {
     const columns = this.props.component.component.definition.properties.columns;
     const column = columns.value[index];
 
+    console.log('col', column);
```
Comment (Navaneeth-pk) [inline frontend/src/Editor/Inspector/Components/Table.jsx:564]: Please remove.

File: frontend/src/_styles/theme.scss [hunk 2]
```diff
@@ -1447,7 +1447,12 @@ border-radius: 4px;
   .has-actions {
     margin: 0;
   }
-
+.text-wrapper{
```
Comment (Navaneeth-pk) [inline frontend/src/_styles/theme.scss:1450]: Indentation looks incorrect here.

**Comment by sherfin94** [comment]
The text wrap option is shown for column types for which it does not apply, show it only for default and string

image

**Comment by sherfin94** [comment]
I think the name of the property should be `overflow` and options should be `wrap`, `hide` and `scroll`

**Comment by sherfin94** [comment]
While using the scroll option, a scroll bar should be shown when that cell is hovered and the cell is overflown. Right now only mac users will be able to use the scroll, correct me if I'm wrong.

**Comment by sherfin94** [comment]
Using the `hide` option destroys the table's structure: 
image

**Review by sherfin94 — CHANGES_REQUESTED** [review]
Commented some issues.

**Comment by stepinfwd** [comment]
issues resolved

**Comment by sherfin94** [comment]
The scroll and wrap options work correctly, however the hide option hides the entire column instead of just hiding the part of the cell that overflew, please check:


https://user-images.githubusercontent.com/20333972/154009124-4e7131e4-87a0-484b-85bf-e3baa643fbef.mov

**Comment by sherfin94** [comment]
Also, the `Text` column type does not require overflow options, it is specifically designed to contain big volumes of text

**Review by sherfin94 — CHANGES_REQUESTED** [review]
Few changes requested, otherwise everything looks fine.

File: frontend/src/Editor/Components/Table/Table.jsx [hunk 3]
```diff
@@ -760,6 +772,7 @@ export function Table({
             ),
             Cell: ({ row }) => (
               <div className="d-flex flex-column align-items-center">
+                ]
```
Comment (sherfin94) [inline frontend/src/Editor/Components/Table/Table.jsx:775]: @stepinfwd Dude what's this 😅 

image

Comment (sherfin94) [reply frontend/src/Editor/Components/Table/Table.jsx:775]: Other than this I don't see any issue, remove this and merge the latest develop. And then let's merge it.

Comment (stepinfwd) [reply frontend/src/Editor/Components/Table/Table.jsx]: updated

**Review by sherfin94 — APPROVED** [review]

**Pull request merged**